packages/imperative/src/rest/src/session/ConnectionPropsForSessCfg.ts

```diff
@@ -366,7 +386,7 @@ export class ConnectionPropsForSessCfg {
                 let answer;
                 while (answer === undefined) {
                     const hideText = profileSchema[value]?.secure || this.secureSessCfgProps.has(value);
-                    let promptText = `${this.promptTextForValues[value]} ${serviceDescription}`;
+                    let promptText = `${this.promptTextForValues[value] ?? `Enter your ${value} for`} ${serviceDescription}`;
```

Comment:
SonarCloud is flagging this line for nested template literals. Thoughts on making these changes to resolve the issue?

```suggestion
                    const valuePrompt = this.promptTextForValues[value] ?? `Enter your ${value} for`;
                    let promptText = `${valuePrompt} ${serviceDescription}`;
```

Reply:
resolved